stock_return_request/hooks.py

```diff
@@ -41,7 +41,7 @@ def pre_init_hook(cr):
 
 def post_init_hook(cr, registry):
     """Set moves returnable qty on hand"""
-    with api.Environment.manage():
+    if SUPERUSER_ID:  # Fake if for not having a big diff
```

Comment:
@pedrobaeza Is the condition justified for a 'diff' problem ?

Reply:
I think so for avoiding 40 lines of diff (the + and -).

Reply:
Yes, but what that matters is the result. And the condition is unwanted

Reply:
PK, as you prefer